### ISSUE TITLE
[Suggestion] Canopy Pieces

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#2987]
The current selection of canopy tile pieces is a brilliant start and shows what can be achieved with the brick system. However, it has some shortcomings. Namely, a very thick border on the canopy pieces which obscure your vision, a large (0.5m) starting size and a few missing shapes.
Here's my suggestion on how to address these -
![image](https://user-images.githubusercontent.com/27857711/93793706-18ebf200-fc2f-11ea-88bb-997c2cb8b5ca.png)

Changes -
All shapes start off as 0.25m cubes rather than the current 0.5m cubes
The border has been reduced in thickness by a factor of 2
Hepta and Penta shapes added, as well as the half shapes adjusted to fit the new scale

Here's an example of some of these shapes put together and stretched. In this test the Hepta had some strange geometry when stretched that could cause issues, although I might have misunderstood how your stretch tech works. It's an extremely important shape though so it cannot be forgotten
![image](https://user-images.githubusercontent.com/27857711/93795162-ea6f1680-fc30-11ea-9976-a237dd9af221.png)

**Comment by Garrett-C** [comment]
I also like this, would be nice if there was a version of the wedge without the frame aswell.

**Comment by PopkaAkula** [comment]
This would be a lifesaver on smaller builds

**Comment by TIKIRobo** [comment]
yes

I need this
Please and thanks

**Comment by Midge57** [comment]
Oh _yes_ please

**Comment by Womble-TC** [comment]
Despite the fact propeach told me to post this, I actually think this would be a really good addition for small fighters and stuff.

**Comment by StarrKiss** [comment]
I need this in my life this is amazing seriously- this adresses the weird issue Ihave with glass heptas

**Comment by Tezlz** [comment]
This is definitely needed, as they are not the canopy pieces are a bit awkward to build with due to some missing shapes and due to them not being 0.25x0.25

**Comment by SageThe13th** [comment]
This is great. I also second the desire for frameless wedges.

**Comment by glorbo703** [comment]
cool 😎😎

**Comment by SovietSponge** [comment]
More shapes would be amazing, fighter canopies could get way more creative in shape and as someone who does fighters a lot I'd certianly make use of it.

**Comment by JLAMontoya** [comment]
Definitely need the wedge to be frameless- we can build the frame with bricks, the frame on the brick itself is very obstructive.

**Comment by Vicomt** [comment]
I would love ALL of this. There's only one problem with glass - and that's the interface between blocks. if that can disappear, that would be awesome.

**Comment by SystemGlitch00** [comment]
I have actually wished for this ever since attempting to make a mono-pod from cowboy bepop. Had to end up using regular blocks instead.

**Comment by Claakin** [comment]
Definitely more glass pieces needed, agreed, but also is it possible to make the glass color changeable??
Stained glass Mother Mary Holy War ship here we come!

**Comment by 5URG3** [comment]
I agree, and I'd like most of these shapes as solid plates without glass too, if that is possible.

**Comment by LongLanguidLizard** [comment]
Got my vote. Would love to see some more glass pieces added to the game and this seems like a great way to do that. Nice!

**Comment by Misteropie** [comment]
This it would awesome if we could get this in the game.

**Comment by ZachZent** [comment]
![](https://cdn.discordapp.com/attachments/271723356795961345/759913821090480128/unknown.png)

**Comment by tsunamayo** [comment]
Thanks, thats some stellar 3d presentation ;) And you got the stretch tech right. I think you are right overall, I will digest this and see what I can do... Thanks!

**Comment by WILDCATreactor** [comment]
@Claakin 
> Definitely more glass pieces needed, agreed, but also is it possible to make the glass color changeable??
> Stained glass Mother Mary Holy War ship here we come!

Transparent paint/stickers, just saying.

**Comment by JRL101** [comment]
This i can agree with, Even having these as shapes: Bricks & Canopy, then having the option to make them transparent etc. That material paint for if it is or not. Meshing canopy with bricks can be a task, but having the solid vs glass option would be amazing.
